ccmbioinfo/lfs
# CARDS-1539: Trusted permissions mode: Users outside of the Trusted users group should not be seeing any errors in the Dashboard, Forms and Variants

- move the code in a new module separate from proms
- only add the dashboard view extension when the trusted permissions scheme is enabled

To test:
- start with `-P cards4lfs --permissions open`
- log in with `admin`, make sure all the sidebar links work correctly
- add a new user, log in with it, make sure all the sidebar links work correctly
- stop, remove `.cards-data`
- start with `-P cards4lfs --permissions trusted`
- log in with `admin`, make sure all the sidebar links work correctly
- add a new untrusted user, log in with it, make sure the sidebar is empty and "wait to be approved" is displayed
- add a new trusted user, log in with it, make sure the sidebar all the sidebar links work correctly
- stop, remove `.cards-data`
- start with `-P cards4proms --permissions trusted`
- log in with `admin`, make sure all the sidebar links work correctly
- add a new untrusted user, log in with it, make sure the sidebar is empty and "wait to be approved" is displayed
- add a new trusted user, log in with it, make sure the sidebar all the sidebar links work correctly
